wesnoth-manual.po

```diff
@@ -560,12 +554,12 @@ msgid ""
 "you choose not to use a unit at all during a scenario it is carried over to "
 "the next, so you don&#8217;t lose units you don&#8217;t use."
 msgstr ""
-"Tažení jsou sekvence bitev, propojených příběhem. Typické tažení se skládá s "
-"přibližně 10-20 scénářů. Hlavní výhodou tažení je, že Vám umožňují vyvíjet "
-"Vaši armádu. Po zakončení každého scénáře jsou jednotky uschovány pro "
-"použití v příštích scénářích, kde je můžete, ale nemusíte přivolat. Pokud je "
-"nepřivoláte, zůstanou i tak nadále k dispozici ve všech dalších scénářích "
-"tažení."
+"Tažení jsou sekvence bitev, propojených příběhem. Typické tažení se skládá "
```

Comment:
zas těsný přívlastek a navíc bych řekl, že spíš rozvíjí hlavní (i když ošklivé) jméno (sekvence), bitev bude asi přívlastek neshodný, ten bych nerozvíjel
no fuj, to je složité, snažím se jen srozumitelně popsat, co mi říká cit
```suggestion
"Tažení jsou sekvence bitev propojené příběhem. Typické tažení se skládá "
```

Reply:
Jo, cítím to stejně.